scripts/exgfs_atmos_gempak_gif_ncdc_skew_t.sh

```diff
@@ -68,11 +68,11 @@ export filesize=$( ls -l rdbfmsua.out | awk '{print $5}' )
 ################################################################
 
 if (( filesize > 40 )); then
-    cp rdbfmsua.out "${COM_ATMOS_GEMPAK_UPPER_AIR}/${RUN}.${cycle}.msupperair"
-    cp sonde.idsms.tbl "${COM_ATMOS_GEMPAK_UPPER_AIR}/${RUN}.${cycle}.msupperairtble"
+    cp rdbfmsua.out "${COMOUT_ATMOS_GEMPAK_UPPER_AIR}/${RUN}.${cycle}.msupperair"
+    cp sonde.idsms.tbl "${COMOUT_ATMOS_GEMPAK_UPPER_AIR}/${RUN}.${cycle}.msupperairtble"
```

Comment:
Where does `COMOUT_ATMOS_GEMPAK_UPPER_AIR` get declared?

Reply:
It's set in `jobs/JGFS_ATMOS_GEMPAK_NCDC_UPAPGIF`:

https://github.com/NOAA-EMC/global-workflow/blob/5789de3caa51fa4d38715e3742fb19f1954dbf93/jobs/JGFS_ATMOS_GEMPAK_NCDC_UPAPGIF#L35-L42

This could probably be made a bit more intuitive, though.  I will make that change.

Reply:
Ah, yeah I searched for uppercase `UPPER_AIR` and didn't see it. Thanks!

Reply:
Done.  And on second look, the COMIN directories are not used, so I remove those declarations.

Reply:
Great, thanks @DavidHuber-NOAA !